src/LiquidityPool.sol

```diff
@@ -150,8 +166,8 @@ contract LiquidityPool is Auth {
     }
 
     /// @dev request share redemption for a receiver to be included in the next epoch execution. Shares are locked in the escrow on request submission
-    function requestRedeem(uint256 shares) public {
-        investmentManager.requestRedeem(shares, msg.sender);
+    function requestRedeem(address owner, uint256 shares) public withTokenApproval(owner, shares) {
```

Comment:
```suggestion
    function requestRedeem(uint256 shares, address owner) public withTokenApproval(owner, shares) {
```

Reply:
yeah, I made it consistent with the modifier, which is consistent with the ERC20 params order :D